lib/errors.js

```diff
@@ -19,25 +19,25 @@ function explain (ok, message, assert, stackStartFunction, actual, expected, top
   const err = new AssertionError({ stackStartFunction, message, operator: assert, actual, expected })
   stackScrub(err)
   if (top) {
-    err.at = {
+    const at = {
       line: top.getLineNumber(),
       column: top.getColumnNumber(),
       file: top.getFileName()?.replace(/\?cacheBust=\d+/g, '')
     }
```

Comment:
if this is just internal we should just do `const line = ... const column = ... ` etc